### ISSUE TITLE
git submodule miniChRIS-docker, miniChRIS-podman, and charts to proper directory-names.

### ISSUE THREAD

**Issue by chhsiao1981** [FNNDSC/ChRIS_ultron_backEnd#552]
`make.sh` is a 1990s-style deployment setup.

In addition, I don't think it's good to put several `.sh` in the root-dir.

Thanks to @jennydaman's modern deployment setup in

[miniChRIS-docker](https://github.com/FNNDSC/miniChRIS-docker)
[miniChRIS-podman](https://github.com/FNNDSC/miniChRIS-podman)
[charts](https://github.com/FNNDSC/charts)

Perhaps we can:
* git submodule miniChRIS-docker docker
* git submodule miniChRIS-podman podman
* git submodule charts helm
* move the original bash-scripts to `make/`:

I understand that we do want to have some simple bash-scripts for deployment.
I would propose to have these scripts in `deploy/`:
* `deploy/install-docker.sh`
* `deploy/uninstall-docker.sh`
* `deploy/install-podman.sh`
* `deploy/uninstall-podman.sh`
* `deploy/install-helm.sh`
* `deploy/uninstall-helm.sh`
* `deploy/install-make.sh`
* `deploy/uninstall-make.sh`

**Comment by jennydaman** [comment]
Hi @chhsiao1981 100% agree that `./make.sh` should be rewritten for the modern era.

The various ways to run _ChRIS_ are described here. https://chrisproject.org/docs/run

tl;dr `./make.sh` is for backend development only, and primarily used by @jbernal0019 himself. The other methods: miniChRIS-docker, miniChRIS-podman, charts, and the documentation you find on https://chrisproject.org/docs, are suitable for "general users" and maintained by me.
